### PR TITLE
Fix references to Estuary and Textile, per Eva's comment.

### PR DESCRIPTION
**Pull request by robertagora**
Removes references to Estuary and Textile Powergate.

### CODE REVIEW

File: basics/project-and-community/filecoin-faqs.md
Comment (smagdali) [inline basics/project-and-community/filecoin-faqs.md:1]: I think 'on an FPS' should be on IPFS?

File: storage-providers/skills/sales.md
Comment (smagdali) [inline storage-providers/skills/sales.md:1]: slingshot is also being deprecated...

**Review by smagdali — COMMENTED** [review]
couple of small fixes.

let's deploy but we really need to validate all the items on this page.

File: basics/project-and-community/filecoin-faqs.md
Comment (robertagora) [reply basics/project-and-community/filecoin-faqs.md:1]: Fixing

File: storage-providers/skills/sales.md
Comment (robertagora) [reply storage-providers/skills/sales.md:1]: Fixing by removing the reference

**Comment by robertagora** [comment]
This is ready to deploy @smagdali --we may want to talk about removing the section that originally referenced slingshot all together.

**Review by smagdali — APPROVED** [review]

**Pull request merged**